src/ni/vsbuild/steps/LvVITesterStep.groovy

```diff
@@ -0,0 +1,18 @@
+package ni.vsbuild.steps
+
+import ni.vsbuild.BuildConfiguration
+
+class LvVITesterStep {
```

Comment:
`LvVITesterStep` still needs to inherit from `LvStep`, just not `LvProjectStep`. I can't imagine a build passing if you ran one with this change.

Reply:
So like this, right? `class LvVITesterStep extends LvStep {`

Reply:
Yes

Reply:
![image](https://user-images.githubusercontent.com/33259264/56241885-9edc3480-605c-11e9-9813-73effd4d4763.png)

Here are the results, build #41 contains
```
class LvVITesterStep {
```
build #42 and #43 contains
```
class LvVITesterStep extends LvStep {
```